game-app/game-core/src/main/java/org/triplea/util/PointFileReaderWriter.java

```diff
@@ -44,7 +44,7 @@ private PointFileReaderWriter() {}
   public static Map<String, Point> readOneToOne(final Path input) throws IOException {
     checkNotNull(input);
 
-    final Map<String, Point> mapping = new HashMap<>();
+    final Map<String, Point> mapping = new LinkedHashMap<>();
```

Comment:
Class javadoc should be updated to note that we preserve polygon ordering  as read from the polygons.txt file. 
That could still be a confusing comment and we'll need to make sure this class is only used to read polygon information. If the class is generic, the comment should be generic too I suppose. Regardless, a developer may wonder why we are not using `LinkedHashMap`

Reply:
I understand the desire to keep an unpublicized contract.

I do not think that is what I asked for. To get on the same page, my perspective is that javadocs are only read by developers, and only read while they are reading source code. 

If order matters, one would expect a `List` to be used and not a `Map`. I can't speak to whether that is a worthwhile update, I would suspect no. Failing that though, even a quick line comment to a developer telling them why it would be a bug to switch this back to `hashMap`, would be helpful.